notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -38,7 +38,7 @@
     "WORKSPACE_DIR = WORKSPACE_ROOT / WORKSPACE_NAME\n",
     "WORKSPACE_DIR.mkdir(exist_ok=True)\n",
     "\n",
-    "SOURCE_DOCUMENT = None # to process a specific document, set its path here; otherwise, the entire source documents repository will be used\n",
+    "SOURCE_DOCUMENT = \"/home/alina/dev/pdfs/merged_cell_pdfs/BofA_InterestChecking_en_ADA.pdf\" # to process a specific document, set its path here; otherwise, the entire source documents repository will be used\n",
```

Comment:
```suggestion
    "SOURCE_DOCUMENT = None # to process a specific document, set its path here; otherwise, the entire source documents repository will be used\n",
```

Reply:
oops missed this! thank you